tool/generator/lib/src/projection/winrt/declarations/comobject.dart

```diff
@@ -0,0 +1,31 @@
+import '../../../../generator.dart';
+
+class COMObjectMethod extends WinRTMethodProjection {
```

Comment:
I'm a bit worried about the name confusion with `ComMethodProjection`. We may need something more precise here, like `WinRTMethodReturningComObjectProjection`. I know it's a long name, but we need some kind of convention.

Reply:
Yeah, you're right. It does cause some confusion with other classes.

I renamed them based on your suggestions (e.g. `WinRTMethodReturningXXProjection`, `WinRTGetPropertyReturningXXProjection`, and `WinRTSetPropertyReturningXXProjection`).